src/picker.common.ts

```diff
@@ -96,31 +96,34 @@ export class PickerField extends TextField implements TemplatedItemsView {
 
         this.applyCssScope(this._page.actionBar);
 
-        let actionItem = new ActionItem();
-        actionItem.text = "Close";
-        actionItem.on(Button.tapEvent, (args: ItemEventData) => {
-            this.closeCallback(undefined, undefined);
-        });
+        const isNavigationButton = this._page.android && this.androidCloseButtonPosition === "navigationButton";
```

Comment:
This will work as expected but we have a built-in module that can provide information regarding the platform: https://docs.nativescript.org/ns-framework-modules/platform . If you don't mind I think it will look better than using the `_page.android` in the condition.

Reply:
I have handled that.